pkg/doc/presexch/README.md

```diff
@@ -0,0 +1,104 @@
+# Presentation Exchange
+
+This document shows how to use [Presentation Exchange](https://identity.foundation/presentation-exchange) by examples.
+Code examples can be found [here](example_test.go).
+
+1. This example demonstrates `predicate` and `limit_disclosure` usage.
+   The `presentation definition` below requires field `age` to be greater or equal to 18.
+   Also, we have `limit_disclosure=true` which requires that output data is limited to the entries specified in the `fields` property.
+   The `predicate` means that the result should be expressed as a boolean value.
```

Comment:
@soluchok 

> The `predicate` means that the result should be expressed as a boolean value.

This is a misunderstanding of the spec. The predicate is just criteria that the credential submitted in the presentation must satisfy. It is _not_ a transformation function.